page.php

```diff
@@ -36,6 +36,22 @@
 $page_meta_data = get_post_meta( $post->ID );
 $page_meta_data = array_map( 'reset', $page_meta_data );
 
+if ( ! is_tag() ) {
```

Comment:
This works because `is_tag` still returns true.

The fact it does is not as straightforward as it seems. In `tag.php` we [re-assign the "queried object"](https://github.com/greenpeace/planet4-master-theme/blob/f79fda3337458efdc60a52c05bb8b927c22d93c0/tag.php#L26-L29).

This results in WordPress considering the page as being the object that is rendered (e.g. the admin UI will point to this page instead of the tag). But doing so doesn't re-assign `$wp-query->is_tag`, so our check still works.